.github/workflows/deploy-dev.yaml

```diff
@@ -0,0 +1,16 @@
+name: Deploy to Dev Environment
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: mbta/actions/deploy-lambda
+        with:
+          role-to-assume: ${{ secrets.AWS_DEV_ROLE }}
+          lambda-function: elevator-hotline-dev
+          slack-webhook-url: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
Can you make sure these env vars/secrets (and the equivalent prod ones) are configured in the repo settings?

Reply:
done